pkgs/build-support/trivial-builders/default.nix

```diff
@@ -471,23 +475,36 @@ rec {
   symlinkJoin =
     args_@{ name
     , paths
+    , stripPrefix ? ""
```

Comment:
Please add documentation for this above, modifying the original text as needed to weave in the use case.